src/main/java/ch/uzh/ifi/hase/soprafs24/service/SessionService.java

```diff
@@ -125,4 +133,8 @@ public Session endSession(Long teamId) {
         endedSession.getStartDateTime(), endedSession.getEndDateTime());
     return endedSession;
   }
+
+  public boolean isSessionExpired(Session session) {
+    return session.getStartDateTime().plusHours(24).isBefore(LocalDateTime.now());
+  }
```

Comment:
helper methods should be private or protected (expose minimal interface!)

Reply:
I changed it to Privat @5v3nn